app/javascript/components/applications/application_form/ApplicationLanguageSection.js

```diff
@@ -11,8 +11,14 @@ const ApplicationLanguageSection = ({ formApi, editValues }) => {
     <div className='border-bottom margin-bottom--2x'>
       <div className='row'>
         <div className='small-4 columns'>
-          <label>Language Submitted In</label>
-          <Select field='application_language' options={applicationLanguageOptions} />
+          <Field.Select
+            label='Language Submitted In'
+            blockNote='(required)'
+            id='application_language'
+            field='application_language'
+            errorMessage={(_, error) => error}
```

Comment:
Seems like we could cover this if we had a test that tried to submit without filling out any of the required fields. We might already have a test for some of them that you could add this too

Reply:
I'm almost done with the test for it.